fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanTransactionsApiResourceSwagger.java

```diff
@@ -164,6 +164,22 @@ private GetLoanTransactionRelation() {}
         public String reversalExternalId;
         @Schema(example = "[2012, 5, 18]")
         public LocalDate reversedOnDate;
+        @Schema(example = "1000.00")
+        public Double netDisbursalAmount;
+        @Schema(example = "240.00")
+        public Double principalPortion;
+        @Schema(example = "23.90")
+        public Double feeChargesPortion;
+        @Schema(example = "12.80")
+        public Double penaltyChargesPortion;
+        @Schema(example = "33.00")
+        public Double overpaymentPortion;
+        @Schema(example = "55.50")
+        public Double unrecognizedIncomePortion;
+        @Schema(example = "100.00")
+        public Double outstandingLoanBalance;
+        @Schema(example = "[2012, 5, 18]")
+        public LocalDate possibleNextRepaymentDate;
```

Comment:
possibleNextRepaymentDate has nothing to do with Loan transaction. Please remove!